### PR TITLE
feat: Built Python CASSANDRAtoGCS template for Dataproc

### PR DESCRIPTION
**Pull request by saumyasinha-google**

### CODE REVIEW

**Comment by vanshaj-bhatia** [comment]
@saumyasinha-google I can see some unwanted files, please remove them from the branch.

**Comment by saumyasinha-google** [comment]
> @saumyasinha-google I can see some unwanted files, please remove them from the branch.

Removed all the unwanted files.

File: python/dataproc_templates/cassandra/cassandra_to_gcs.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        .config(constants.SQL_EXTENSION, constants.CASSANDRA_EXTENSION)
+        .config(f"spark.sql.catalog.{catalog}", constants.CASSANDRA_CATALOG)
+        .config(f"spark.sql.catalog.{catalog}.spark.cassandra.connection.host",input_host)
+        .getOrCreate())
+ 
+       # Read
+       if(not query):
+           input_data = spark.read.table(f"{catalog}.{input_keyspace}.{input_table}")
+       else:
+           input_data = spark.sql(query)
```
Comment (vanshaj-bhatia) [inline python/dataproc_templates/cassandra/cassandra_to_gcs.py:147]: As per the current design, either query or {input_keyspace}.{input_table} would be required for execution. That means, if the user provides query, then {input_keyspace}.{input_table} are not to be provided. This is not handled correctly as {input_keyspace}.{input_table} are set as required properties.

**Review by vanshaj-bhatia — CHANGES_REQUESTED** [review]

File: python/dataproc_templates/cassandra/cassandra_to_gcs.py (hunk 1 above)
Comment (saumyasinha-google) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:148]: Modified required properties for keyspace and table to be true only when query is None.
Thanks for pointing that out!

Comment (vanshaj-bhatia) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:148]: Please update README as well.

Comment (vanshaj-bhatia) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:148]: @saumyasinha-google What would happen if all three params: table, keyspace and query is provided by the user?

Comment (saumyasinha-google) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:148]: The template will run fine if all three will be provided. However keyspace and table name won't be required properties in such a case but even if they are provided template will run fine.

Comment (vanshaj-bhatia) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:148]: Ideally, it should fail and prompt the user to either provide query or table and keyspace. Please make the necessary changes.

Comment (saumyasinha-google) [reply python/dataproc_templates/cassandra/cassandra_to_gcs.py:162]: Made the changes.

**Review by vanshaj-bhatia — APPROVED** [review]
LGTM

**Pull request merged**